test/integration/install/test_install.py

```diff
@@ -360,7 +360,7 @@ def test_install_tarantool_in_docker(tt_cmd, tmp_path):
     # Check tarantool glibc version.
     out = subprocess.getoutput("objdump -T " + os.path.join(tmp_path, "bin", "tarantool") +
                                " | grep -o -E 'GLIBC_[.0-9]+' | sort -V | tail -n1")
-    assert out == "GLIBC_2.27"
+    assert out == "GLIBC_2.34"
```

Comment:
It does not clear, why we need checks `glibc` version of `Tarantool`? What depends on it?

Reply:
I don't know, so let just update it and fix a problem later.